frontend/src/routes.js

```diff
@@ -12,152 +10,166 @@ const Loadable = (Component) => (props) =>
   );
 
 // Authentication pages
-const Login = Loadable(lazy(() => import('./views/authentication/Login')));
+const Login = Loadable(
+  lazy(() => import('./modules/Misc/views/authentication/Login'))
+);
 
 // Error pages
-const NotFound = Loadable(lazy(() => import('./views/NotFound')));
+const NotFound = Loadable(lazy(() => import('./modules/Misc/views/NotFound')));
 
 const OrganizationList = Loadable(
-  lazy(() => import('./views/Organizations/OrganizationList'))
+  lazy(() => import('./modules/Misc/views/Organizations/OrganizationList'))
 );
 const OrganizationView = Loadable(
-  lazy(() => import('./views/Organizations/OrganizationView'))
+  lazy(() => import('./modules/Misc/views/Organizations/OrganizationView'))
 );
 const OrganizationCreateForm = Loadable(
-  lazy(() => import('./views/Organizations/OrganizationCreateForm'))
+  lazy(() =>
+    import('./modules/Misc/views/Organizations/OrganizationCreateForm')
+  )
 );
 const OrganizationEditForm = Loadable(
-  lazy(() => import('./views/Organizations/OrganizationEditForm'))
+  lazy(() => import('./modules/Misc/views/Organizations/OrganizationEditForm'))
 );
 const EnvironmentCreateForm = Loadable(
-  lazy(() => import('./views/Environments/EnvironmentCreateForm'))
+  lazy(() => import('./modules/Misc/views/Environments/EnvironmentCreateForm'))
 );
 const EnvironmentEditForm = Loadable(
-  lazy(() => import('./views/Environments/EnvironmentEditForm'))
+  lazy(() => import('./modules/Misc/views/Environments/EnvironmentEditForm'))
 );
 const EnvironmentView = Loadable(
-  lazy(() => import('./views/Environments/EnvironmentView'))
+  lazy(() => import('./modules/Misc/views/Environments/EnvironmentView'))
```

Comment:
I am pretty much sure that `Environment` will never be or need to be a module. It's a central piece of the application. `Organization` is too. There might be others. 
By this, I mean that there are some pieces that are unplugable. 

If we put `Environment` and `Organization` into modules we need to deal with a lot of inter-module dependencies

Reply:
My opinion on this and Raul (senior SDE from my team) agreed with me here, is that we don't need to use file structure to force the users into some behaviour, we could agree on some kind of contribution guid, we could put a Github action to make sure they are not updating some part that they should not, and so on...

This part in the planning is left for future discussion, we don't need to finalise it now. We agreed to modularize the application gradually anyway. So I think in the future things will be more clear as to what works and what does not.

Reply:
My biggest concerns here is only the folder where the changes placed. It seems strange for me that the environment is in modules and also in Misc (since it's a central part).
In the planning, I suggested using core for this reason. First, everything would be core and then we would split onto modules. Now, we have to do the other way around. 
Basically, that's not a blocker, so if the other are agree we can merge it

Reply:
it's all the same really, the FE won't be merged into main until the modules are finished anyway